.github/workflows/automerge.yml

```diff
@@ -0,0 +1,25 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        if: github.actor != 'dependabot-preview[bot]' && github.actor != 'dependabot[bot]'
+        uses: "pascalgn/automerge-action@v0.6.0"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
1. Why did you change this workflow?
2. dependant is already able to auto merge PRs that are approved and I've even configured the autorebase so all the updates merge themselves as long the test pass. Did you want to change something in particular here?

Reply:
Copied from Spectral, this has nothing to do with dependabot. It’s handy to add when you’re waiting for a branch to update and you’re expecting (or already have) approval. You can add this label and wander off to do something more useful.

Reply:
> this has nothing to do with dependabot

Why did you remove the auto-approve for dependabot then?

Apart from this anyway, I do not really see the value at the moment — this is not a bottleneck in this project. Can we remove/reevaluate?

Reply:
OH I had no idea I removed the dependabot. I saw no .github/workflows folder, so I coped over the one from Spectral and considered it a happy accident that we got `automerge` label support. We started it on Spectral with the goal of moving it over, but if your PR cadence is low enough that it doesn't need to happen then fantastic! I'll put things back as they were.